code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -198,7 +202,12 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 
 /datum/game_mode/dynamic/send_intercept()
 	. = "<b><i>Central Command Status Summary</i></b><hr>"
-	switch(round(threat_level))
+	var/shown_threat
+	if(prob(FAKE_REPORT_CHANCE))
+		shown_threat = rand(1 to 100)
```

Comment:
wtf?

Reply:
![firefox_2021-01-18T12-37-18](https://user-images.githubusercontent.com/35135081/104960832-ec9c7400-5989-11eb-9377-8875ea7641d3.png)

it compiles but...its always 1...WTF?

Reply:
![image](https://user-images.githubusercontent.com/35135081/104960860-00e07100-598a-11eb-9d79-2ede1a6a1a14.png)

WHY DOES THIS COMPILE

Reply:
hypothesis, (1 to 100) makes some sort of internal list of the numbers between 1 and 100, hell maybe even an iterator. rand asks for the first value, sees one, and finishes. Also cringe

Reply:
haha woops wrong language